mail/_mail.html

```diff
@@ -101,6 +101,9 @@
 
 </table>
 
+<p>
+  Besides mailing lists there are also https://github.com/curl/curl/discussions .
```

Comment:
Doing it like this will show the URL in text and not as a link. This is HTML. We want a link with a name I think.

Reply:
Oops. I'll fix it as soon as I get to my desktop computer.